.gitlab-ci.yml

```diff
@@ -3766,25 +3766,31 @@ deploy_cloudfront_invalidate_on_failure:
 #
 # Trigger release pipelines
 #
+
+# The trigger jobs are always run (even on failing pipelines)
+# because there's no way to retry a trigger job once it gets skipped
+# because of a pipeline failure, even when retrying the pipeline.
 trigger_release_7:
   <<: *run_only_when_triggered_on_tag_7
   stage: trigger_release
   needs: ["deploy_cloudfront_invalidate_on_success"]
   variables:
-    RELEASE_VERSION: $RELEASE_VERSION_7
+    RELEASE_VERSION: $RELEASE_VERSION_7-1
```

Comment:
Why the `-1`? If this is needed, shouldn't it be added by the script in `agent-release-management`?

Reply:
The `agent-release-management` pipeline shouldn't need to assume anything about the package it has to handle, so I prefer giving it the full package version.